addon/components/sharing-icons-popover/component.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+import layout from './template';
+import config from 'ember-get-config';
+import Analytics from '../../mixins/analytics';
+/**
+ * @module ember-osf
+ * @submodule components
+ */
+
+/**
+ * Display a share icon that pops out sharing options
+ * @class sharing-icons-popover
+ *
+ * Sample usage:
+ * {{sharing-icons
+ *     title=title
+ *     description=description
+ *     hyperlink=hyperlink
+ *     metricsExtra=metricsExtra
+ * }}
+ */
+export default Ember.Component.extend(Analytics, {
+    layout,
+    host: config.OSF.url,
+    i18n: Ember.inject.service(),
```

Comment:
Not sure if it's convention, but listing injected services before properties is nice.